### DISCUSSION TITLE
$5,000 NFT Marketplace Acquisition for Darwinia.

### DISCUSSION THREAD

**Discussion by johneggleston** [ringecosystem/discussion#6] [General]
Dear Darwinia Team,

I am writing to propose the acquisition of LEDA, a fully functional NFT marketplace, for the sum of $5,000 that has been in my possession for nearly 2 years now. LEDA has been built to MVP stage and is ready to be integrated into a larger ecosystem such as Darwinia, providing a valuable infrastructure that supports and complements the ecosystem nicely. Darwinia will have the freedom to rebrand as they please, but are also free to keep the name LEDA if they please. You can view the marketplace for yourselves here: ledav2.jup.io(https://ledav2.jup.io/)

The marketplace MVP supports collections, unique user profiles, and other social features such as liking and sharing NFTs. It has been optimised for scalability and interoperability, offering a streamlined and user-friendly platform for artists, creators, and collectors to interact. 

By purchasing an NFT marketplace already at MVP stage, Darwinia can significantly reduce development time and cost and enter the growing NFT economy with minimal additional development costs. 

For $5,000, you guys will receive all the MVP-stage code, documentation, and rights to the marketplace. I would also be happy to provide support during the transition period to facilitate seamless integration into Darwinia’s ecosystem.

I would be happy to discuss this proposal in more detail and answer any questions you might have. LEDA represents a unique opportunity to expand your ecosystem in a meaningful way, and I am confident that, under Darwinia’s leadership, the marketplace can grow into a vital component of the ecosystem.

Please let me know a suitable time for further discussion if you are interested. 

Thank you for your consideration, 
John Eggleston, DCDAO Ambassador.

**Comment by hackfisher** [comment]
Hi John,

Some points here, hope this helps:

1. I’d like to remind that there is an ongoing grant in the community also related to NFT market application: [sublayer.md](https://github.com/ringecosystem/collaboration/blob/master/grant/projects/sublayer.md). After multiple follow-ups, the team submitted a prototype on the Koi testnet last week, available for reference here: [sublayer.art](https://sublayer.art/).

2. Regarding the acquisition of LEDA, An acquisition usually implies the takeover of the company and its team as a whole, along with continued development. And it is unlikely that the acquiring party would hire a separate team to take over,  Simply purchasing the source code wouldn't be meaningful, and a budget of 5K seems insufficient. The application would require ongoing maintenance, promotion, and operational support. RingDAO currently lacks the resources and a prepared arm entity to carry out such an acquisition, and this proposal also conflicts with our current strategy of strict cost control and expense reduction.

3. My suggestion for a potentially better approach for you is to consider talking to the Sublayer team to see if they are interested, with the condition that you continue developing and contributing to the merged NFT market application. Perhaps funding could come from donations or investments from individuals and companies within the community.
